recipes/robotlab/meta.yaml

```diff
@@ -18,7 +18,9 @@ build:
   entry_points:
     - robotlab = robotlab.labapp:main
     - robotlab-extension = robotlab.labextensionapp:main
+    - robotlab-shortcuts = robotlab.shortcuts:make_shortcuts
     - robotlab-examples = robotlab.examples:copy_robotkernel_examples
+    - robotlab-tutorial = robotlab.tutorial:copy_robotkernel_tutorial
```

Comment:
yeah, this is probably the problem! it was falling back to the `setuptools` ones, which generate scripts which don't work so well with conda.